dags/resources/stages/enrich/schemas/transactions.json

```diff
@@ -90,5 +90,25 @@
         "type": "STRING",
         "mode": "REQUIRED",
         "description": "Hash of the block where this transaction was in"
+    },
+    {
+        "name": "max_fee_per_gas",
+        "type": "INT64",
+        "description": "Total fee that covers both base and priority fees"
+    },
+    {
+        "name": "max_priority_fee_per_gas",
+        "type": "INT64",
+        "description": "Fee given to miners to incentivize them to include the transaction"
+    },
+    {
+        "name": "transaction_type",
+        "type": "INT64",
+        "description": "An envelope for future transaction types"
+    },
+    {
+        "name": "effective_gas_price",
+        "type": "INT64",
+        "description": "The amount of gas used by this specific transaction alone (a replacement for gasUsed field)"
```

Comment:
Is it the amount of gas or the gas price?

Would this be correct: `gas_cost = receipt_gas_used * receipt_effective_gas_price`?